docs/maya/simple_setup.md

```diff
@@ -317,3 +317,46 @@ Refer to this [section](simshape#muscle-activations) to see how to use Muscle Pa
 </figure>
 
 In the attribute editor of the AdnSimshape deformer, under the *Muscles Activation* section, the *Plug Values* will be enabled as a new valid *Activation Mode* option. To better visualize activations, press the ![AdnMuscle](images/adn_simshape_debugger.png){style="width:4%"} shelf button or go to AdonisFX Menu > Simshape (on the *Edit* group) > *Activations Debugger*.
+
+## AdnSkinMerge
+
+To create a basic scenario using the AdnSkinMerge deformer, start with a scene with the following elements:
+
+  - An animation mesh with deformation.
+  - A skin mesh with an AdnSkin deformer applied and configured.
+  - A final mesh without animation or deformation.
+
+The AdnSkinMerge deformer will get applied to the final mesh which will combine the animation and simulation meshes.
+
+<figure>
+  <img src="images/setup_skin_merge_0.png"> 
+  <figcaption><b>Figure 32</b>: Basic setup for skin merge (final mesh and animation mesh are the same, but final mesh has no deformation).</figcaption>
+</figure>
+
+### Create Deformer
+
+To create the AdnSkinMerge deformer press the ![AdnSkinMerge](images/adn_skin_merge.png){style="width:4%"} shelf button or go to *AdonisFX Menu* > *Deformers* (on the *Create* group) > *Skin Merge*.
+
+With this action the Create Skin Merge UI will open, allowing to add all the required elements to create the deformer. To add the required meshes select the mesh in the scene and press the corresponding *Add Selected* button. You may also set up a custom name and initialization time in this window before creating the deformer.
+
+When everything has been properly set up, press the *Create* button to create the AdnSkinMerge deformer.
+
+<figure>
+  <img src="images/setup_skin_merge_1.png"> 
+  <figcaption><b>Figure 33</b>: Create Skin Merge window with corresponding meshes added.</figcaption>
+</figure>
+
+### Paint Weights
+
+Once we have the AdnSkinMerge deformer created we will paint its weights to blend the animation and simulation meshes into the final mesh. Similar to AdnSimshape, this deformer uses the Maya Paint tool to setup and paint its paintable weight attributes.
+
+With this deformer we will paint the *Blend* attribute with high values over the areas where we want the simulated meshes results to be more visible.
+
+In areas near the edges between the simulation and animation meshes, paint with medium values for a better transition between meshes. We must be cautious not to paint outside the limits of the simulation meshes.
+
+<figure>
+  <img src="images/setup_skin_merge_2.png"> 
```

Comment:
I've added a comment related to "blend_weights" png file. I think we have to smooth more

Reply:
Updated screenshot!